wiki/fr/fr-Getting-Started.md

```diff
@@ -37,7 +37,10 @@ Maintenant que vous avez les bases, c'est parti :
 
 Découvrez aussi : [Jamulus OS !](https://sourceforge.net/projects/jamulus-os/files/JamulusOS/) Exécutez Jamulus sur n'importe quel ordinateur avec une clé USB (uniquement pour les systèmes en 64 bits). Vérifiez la compatibilité de votre matériel et profitez-en !
 
-## Tout est bon ?
+## Matériel recommandé
+
+Voir [la liste de bon matériel connu](Sound-Devices) et la [configuration du matériel](Hardware-Setup).
```

Comment:
"la liste de bon matériel connu" sounds not fine to mee.
I would have worded it this way:
"la liste des bons équipements connus" or "la liste des bons périphériques connus". (possible translations for "Sound Devices", without "sonores" not necessary, i think).

Reply:
We're not translating "_Sound-Devices_" which is the targeted page name here, but the link name which is "_the list of known good hardware_". The main translation of "hardware" in French is "matériel". "Èquipment" could work but is much less usual than "matériel". "Périphérique" would be wrong here since it's the French translation of "device".